### ISSUE TITLE
SideCar.Start not ending

### ISSUE THREAD

**Issue by Arik-Shapiro** [riezebosch/Dapr.Wrapr#4]
Running your 'DaprDemo.Api.IntegrationTests' example the Start never ends it keeps waiting but never finishes, when omitting the 'await' it works (if you actually wait for the sidecar to go up properly).

Also, is there a way to find out dynamically Grpc/Http ports when omitted from Start?

Thanks.

**Comment by riezebosch** [comment]
Hi! I've stumbled on this before and did some work on it. Right now it scans the output to make sure the sidecar is available, but with recent updates on the dapr-side you can let the client wait for the sidecar to be available. Hope to implement this in the coming days.

**Comment by riezebosch** [comment]
If I run the integrations tests of this project with updated dapr and updated dependencies it still seems to work? You already tried to inject a logger and inspect the output?

**Comment by Arik-Shapiro** [comment]
> If I run the integrations tests of this project with updated dapr and updated dependencies it still seems to work? You already tried to inject a logger and inspect the output?

The logs are just showing the side car running.
I'm confused from your code it's waiting for the output of '✅' but it never comes.
when changing it to something like "dapr initialized. Status: Running." it works and stops.
Is the '✅' in an old version of dapr?

**Comment by riezebosch** [comment]
```
dapr --version
CLI version: 1.11.0 
Runtime version: 1.11.2
```

But I'm testing it in the [self-hosted mode](https://docs.dapr.io/operations/hosting/self-hosted/self-hosted-no-docker/), maybe output there is different?

**Comment by jpberset** [comment]
> ```
> dapr --version
> CLI version: 1.11.0 
> Runtime version: 1.11.2
> ```
> 
> But I'm testing it in the [self-hosted mode](https://docs.dapr.io/operations/hosting/self-hosted/self-hosted-no-docker/), maybe output there is different?

The test works on linux, but not on windows. there are no '✅' but only plain text

<img width="643" alt="image" src="https://github.com/riezebosch/Dapr.Wrapr/assets/87026502/09767ef4-c156-4cf6-b87f-f65eee362240">

**Comment by riezebosch** [comment]
Should be fixed with [v1.0.32](https://www.nuget.org/packages/Wrapr/1.0.32).
